src/tabnote.ts

```diff
@@ -466,13 +491,13 @@ export class TabNote extends StemmableNote {
     this.stem.setNoteHeadXBounds(stem_x, stem_x);
 
     if (render_stem) {
```

Comment:
This is a better place to put a check for `this.stem`.

Reply:
Done this way, I am happy to see we agree! :)